recipes/dask-gateway/meta.yaml

```
@@ -0,0 +1,112 @@
+
```

name is undefined, so nothing is running in any of the CI builds (odd that this isn't an error).

```suggestion
{% set name = "dask-gateway" %}
```
```

---

really unclear how that got lost :confused: